src/popup/sass/components/_modal.scss

```diff
@@ -0,0 +1,66 @@
+.modal {
+  display: none;
+  position: fixed;
+  z-index: 10;
+  padding-top: 140px;
```

Comment:
All the below rules can be nested inside `.modal`rule.

Reply:
I have update the PR but for this feedback I am unable to understand how to approach, also these rules are not being repeated.

Reply:
I have added the suggestion below.